src/components/Inputs/TextAreaInput/TextAreaInput.md

```diff
@@ -0,0 +1,25 @@
+### Default
+
+```jsx
+// formChangeHandler gets wired up automatically if using <Form /> component
+const formChangeHandlerStub = () => {}
+
+;<TextAreaInput
+  name="example"
+  labelCopy="TextAreaInput"
+  data-tid="the-textarea-input"
+  formChangeHandler={formChangeHandlerStub}
+  validator={() => ''}
```

Comment:
It would be nice to have a failure case here just to show the field level validation hints.